### PR TITLE
Exm105 R4 Synthea module

### PR DESCRIPTION
**Pull request by okeefm**
A precursor to an EXM105 R4 `fhir-patient-generator` pull request, this PR updates Synthea to contain an EXM105 R4 module.

Ways to test:
* Drop the `stroke_exm_105_r4` module into the `Paste JSON` option on https://synthetichealth.github.io/module-builder/#Untitled
* Checkout the branch `EXM_105_r4` on `fhir-patient-generator`, run `rm -rf synthea/` in that repo, then run `make MEASURE_DIR=EXM_105`.

### CODE REVIEW

**Review by mgramigna — APPROVED** [review]
Looks good

File: src/main/resources/modules/stroke_exm_105_r4.json [hunk 1]
```diff
@@ -0,0 +1,192 @@
+{
```
Comment (tstrass) [inline src/main/resources/modules/stroke_exm_105_r4.json:1]: I ran this twice via `fhir-patient-generator` `EXM_105_r4` branch, and am getting No Population for all patients. Not immediately clear to me why, but are you guys getting patients in measure populations? @okeefm @mgramigna

Comment (okeefm) [reply src/main/resources/modules/stroke_exm_105_r4.json:1]: Did you delete the `synthea/` directory then re-run `make`?

Comment (okeefm) [reply src/main/resources/modules/stroke_exm_105_r4.json:1]: Also, I think there may need to be updates to that branch now that `fhir-bundle-calculator` has updated. I'll make those shortly.

Comment (tstrass) [reply src/main/resources/modules/stroke_exm_105_r4.json:1]: Good call, my bad I forgot `make clean` doesn't remove `synthea`. I got better results this time so 👍

**Review by tstrass — APPROVED** [review]

**Pull request merged**